### DISCUSSION TITLE
Welcome to Jekyll! | The Lambda-Thing

### DISCUSSION THREAD

**Discussion by giscus** [TanguyAndreani/tanguyandreani.github.io#3] [Announcements]
# Welcome to Jekyll! | The Lambda-Thing

You’ll find this post in your _posts directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run jekyll serve, which launches a web server and auto-regenerates your site when a file is updated. Jekyll requires blog post files to be named according to the following format: YEAR-MONTH-DAY-title.MARKUP Where YEAR is a four-digit number, MONTH and DAY are both two-digit numbers, and MARKUP is the file extension representing the format used in the file. After that, include the necessary front matter. Take a look at the source for this post to get an idea about how it works. Jekyll also offers powerful support for code snippets: def print_hi(name) puts "Hi, #{name}" end print_hi('Tom') #=> prints 'Hi, Tom' to STDOUT. Check out the Jekyll docs for more info on how to get the most out of Jekyll. File all bugs/feature requests at Jekyll’s GitHub repo. If you have questions, you can ask them on Jekyll Talk.

https://tanguyandreani.github.io/welcome-to-jekyll.html#2a9c4a1a-6f7c-11ec-af39-21646204cc72

**Comment by TanguyAndreani** [comment]
Hello
